2024/src/data/talks.yaml

```diff
@@ -67,21 +67,33 @@
   title: "LT: Connecting the dots – Web Accessibility through Internationalization and Localization"
   titleJa: "LT: Connecting the dots – Web Accessibility through Internationalization and Localization"
   description: >-
-    Talk Title: Connecting the Dots: Web Accessibility through Internationalization and Localization
+    Talk Title:  Connecting the Dots: Web Accessibility through Internationalization and Localization
+    
+
+    Overview: 
+    
+    In today’s globalized world, web products need to be accessible to users of diverse languages, cultures, and abilities. This talk will explore how internationalization (i18n) and localization (l10n) work hand-in-hand with web accessibility standards to create inclusive, globalized digital experiences.
 
-    Overview: In today’s globalized world, web products need to be accessible to users of diverse languages, cultures, and abilities. This talk will explore how internationalization (i18n) and localization (l10n) work hand-in-hand with web accessibility standards to create inclusive, globalized digital experiences.
 
     Attendees will gain insights into:
 
-    Internationalization: The technical foundations necessary for adapting content for any language or culture, including character encoding, RTL/LTR design, and internationalized code structures.
-    Localization: The process of tailoring a product to meet specific cultural, linguistic, and regulatory requirements, such as translating content, modifying UI elements, and adapting formatting conventions.
-    Web Accessibility: Ensuring that web content is robust and accessible to users with disabilities across different locales, focusing on the intersections between i18n, l10n, and accessibility standards (WCAG).
+
+    #Internationalization: The technical foundations necessary for adapting content for any language or culture, including character encoding, RTL/LTR design, and internationalized code structures.
```

Comment:
I think with `#` you want to make markdown headlines but the result is that the `#Internationalization` is shown in the browser. Further `#` is a `h1` and usually at least a `##` (`h2`) should be used. In other places we use a `h2`, so it should probably look like`## Internationalization` (etc.)